### ISSUE TITLE
Error: 502 Bad Gateway in Windows 10

### ISSUE THREAD

**Issue by n-falsario** [macor161/atom-package-sync#21]
Hi Everyone, 
I've never had a chance to really use this sync feature as I was never able to successfully login to my Google account to start the sync. Upon first install of the package and restart of Atom, I was given the login window to my Google Account.  When I tried to type in my email and password the popup window, the Atom/package froze and didn't alow me to submit my info. A few startups of Atom after that, the Google window was giving me a "502 Bad Gateway" message.  This sounds like a really useful package for those with Atom installed on multiple computers, including myself. I would hate to see it abandoned.  Any help would be appreciated. Thanks in advance.

**Comment by jovaniedelacruz** [comment]
Noticed this issue as well. Wasn't always an issue. This is a very recent bug. 
Fresh install of Windows 10, Atom, and atom-package-sync results in a 502 Bad Gateway message on initial startup. Can't sign in to Google to sync previous packages.

**Comment by tgardner42** [comment]
I've had this fantastic package installed for quite a while now and I'm not sure when it broke, but my guess is something may be misconfigured on the server hosting atom-package-sync.com. Looking around the other issues, I tried the useragent suggestion in issue #19 as it seemed the most likely to help, but it didn't fix this gateway issue (not terribly surprised). I then tried going to everything from https://api.atom-package-sync.com/authentication/app to https://atom-package-sync.com in Chrome and Firefox and it's throwing the 502 on each page. I seem to recall that https://atom-package-sync.com at least loaded a basic page once upon a time. Not sure if this information is actually helpful, but that's what I'm seeing.
